### DISCUSSION TITLE
Record Position

### DISCUSSION THREAD

**Discussion by yasinkt** [sa-si-dev/virtual-select#185] [Q&A] [answered]
This is my selects
![image](https://user-images.githubusercontent.com/76478814/191906192-cedd5f09-5fcd-4bc7-b60f-4201fc8f530c.png)

 city select after country select 
but

search 'e'
![image](https://user-images.githubusercontent.com/76478814/191906566-d501b166-97d4-454b-9668-098803ad1f8c.png)

search 'ea'
![image](https://user-images.githubusercontent.com/76478814/191906644-3a74ed16-e020-4fff-b50d-0fa17a4d1ea3.png)

finded one record but not showing because record position bottom

**Comment by sa-si-dev** [comment]
Set `dropboxWrapper: 'body'` to fix this problem.

**Reply by yasinkt** [reply]
I've tried but not working. 

![image](https://user-images.githubusercontent.com/76478814/191952310-7e61070c-15f3-497c-aa8f-201a7fbadadf.png)

inside this "virtual-select" Offcanvas(Bootstrap 5)

**Comment by sa-si-dev** [comment, marked as answer]
Please share the code you are using to initiate the plugin.

**Reply by yasinkt** [reply]
i fixed this error the problem is in my code. Thank you very much for your support
